go.mod

```diff
@@ -1,33 +1,98 @@
 module github.com/ramendr/ramen
 
-go 1.16
+go 1.17
 
 require (
 	github.com/aws/aws-sdk-go v1.38.41
 	github.com/backube/volsync v0.3.0
+	github.com/csi-addons/kubernetes-csi-addons v0.3.1-0.20220510101231-848e8b27d395
 	github.com/csi-addons/volume-replication-operator v0.1.0
 	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
-	github.com/go-logr/logr v0.4.0
+	github.com/go-logr/logr v1.2.3
 	github.com/kubernetes-csi/external-snapshotter/client/v4 v4.2.0
-	github.com/onsi/ginkgo v1.16.4
-	github.com/onsi/gomega v1.15.0
+	github.com/onsi/ginkgo v1.16.5
+	github.com/onsi/gomega v1.19.0
 	github.com/open-cluster-management/api v0.0.0-20210527013639-a6845f2ebcb1
-	github.com/operator-framework/api v0.10.0
+	github.com/operator-framework/api v0.14.1-0.20220413143725-33310d6154f3
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.11.0
+	github.com/prometheus/client_golang v1.12.1
 	github.com/prometheus/client_model v0.2.0
 	github.com/stolostron/config-policy-controller v0.0.0-20220112151220-7bd880ac5703
 	github.com/stolostron/governance-policy-propagator v0.0.0-20220125192743-95d49290a318
 	github.com/stolostron/multicloud-operators-foundation v1.0.0-2021-10-26-20-16-14.0.20220119065659-a58b01295a6f
 	github.com/stolostron/multicloud-operators-placementrule v1.2.4-0-20220112-8eedb3f
-	go.uber.org/zap v1.19.0
+	go.uber.org/zap v1.19.1
 	golang.org/x/crypto v0.0.0-20211215153901-e495a2d5b3d3
-	k8s.io/api v0.22.6
-	k8s.io/apimachinery v0.22.6
+	k8s.io/api v0.23.6
+	k8s.io/apimachinery v0.23.6
 	k8s.io/client-go v12.0.0+incompatible
-	k8s.io/component-base v0.22.1
-	k8s.io/kube-openapi v0.0.0-20211109043538-20434351676c
-	sigs.k8s.io/controller-runtime v0.9.7
+	k8s.io/component-base v0.23.5
+	k8s.io/kube-openapi v0.0.0-20211115234752-e816edb12b65
+	sigs.k8s.io/controller-runtime v0.11.2
+)
+
+require (
```

Comment:
Where is this coming from? How to regenerate it in the future?

Reply:
ok, so this is because of [go 1.17](https://go.dev/doc/go1.17#go-command)

Does this mean in our sanity action we need to run `go mod tidy -go=1.17`?